### ISSUE TITLE
Image alternative text presence should be conveyed to the user

### ISSUE THREAD

**Issue by mlewand** [ckeditor/ckeditor5#12268]
## 📝 Provide detailed reproduction steps (if any)

Currently the end user has no idea whether an alternative text is applied to a certain image or not.

1.  Open [full featured example for the latest release](https://ckeditor.com/docs/ckeditor5/35.0.1/examples/builds-custom/full-featured-editor.html).
2.  Focus the first image.
3.  Change alternative text to non-empty value.

### ✔️ Expected result

The fact that the image has an alternative text should be conveyed to the user with UI.

**In addition** to that let's change the button name to "Image text alternative".

### ❌ Actual result

No information is present.

![](https://user-images.githubusercontent.com/5353898/184507589-be814551-18a2-45af-abcb-12cb72f67a79.png)

### ❓ Possible solution

IMO just marking the UI button as `isOn` would be sufficient and consistent with how ImageCaption, TableCaption and the alikes are doing this:

![](https://user-images.githubusercontent.com/5353898/184507517-5a5e24a4-6dab-47c7-b40a-23b2808dc5c0.png)

![](https://user-images.githubusercontent.com/5353898/184507518-a8881408-d0c2-49b8-a2b3-96e0a9f890b8.png)

For describing link state we even go extra mile. Not only we're setting `isOn` to `true` on link button, but there's this little "status" icon in the top-right corner of the image:

![](https://user-images.githubusercontent.com/5353898/184507681-18f1c84a-0de8-4dd2-ad91-bf34cb3ff7e5.png)

## 📃 Other details

*   Browser: Any
*   OS: Any

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by oleq** [comment]
I suppose it makes sense 👍

On a side note, we could also spend some time on the UI icon. This was the best we came up with years ago but maybe we could do a little bit better now. It looks awfully similar to the show/hide icon but it shouldn't.

**Comment by mlewand** [comment]
> On a side note, we could also spend some time on the UI icon. This was the best we came up with years ago but maybe we could do a little bit better now. It looks awfully similar to the show/hide icon but it shouldn't.

That's ok but it's a separate issue.
